# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1326657397 18000
#      Sun Jan 15 14:56:37 2012 -0500
# Branch stable
# Node ID fe2ace7a35da88387bbb409575431ae23ab112aa
# Parent  704f7895eef03008dd79848eb9da4bfb40787d73
Added tag release-3.6.0 for changeset 704f7895eef0

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -53,8 +53,9 @@ 695141f1c05cf1b240592bdd18e7a1503bb2a539
 901d466ee55ac902a875ec0ade6f1eccef0841dc release-3-4-1
 3666e8e6f96e6899b8306d6ea9614aadf0500d67 release-3-4-2
 b0e70a71647b671ebcfa7a79af1ae6d3c0f52065 release-3-4-3
 3781981be535e80d44c85373b8fdaa60ca5cd097 ss-3-5-90
 ff5588774680d4f54567311fc109c8e351950f1c ss-3-5-91
 a737b3fb9c4d89d3694da6b4e623aeee64b212e1 ss-3-5-92
 72aebe6196414e38ef802469ff6e238f914f04c9 rc-3-6-0-0
 64d9f33313cc8c691974bcd123357e24bccbabdc rc-3-6-0-1
+704f7895eef03008dd79848eb9da4bfb40787d73 release-3.6.0
